toqito/nonlocal_games/tests/Game4MUB.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
```

Comment:
Could we add these tests to the existing `test_extended_nonlocal_games.py` file? 
https://github.com/vprusso/toqito/blob/master/toqito/nonlocal_games/tests/test_extended_nonlocal_game.py